tensorflow/python/kernel_tests/self_adjoint_eig_op_test.py

```diff
@@ -78,7 +78,7 @@ def Test(self):
         low=-1.0, high=1.0, size=n * n).reshape([n, n]).astype(dtype_)
     a += a.T
     a = np.tile(a, batch_shape + (1, 1))
-    if dtype_ == np.float32:
+    if dtype_ != np.float64:
```

Comment:
Thanks @martinwicke @rmlarsen for the review. The PR has been updated. Please take a look.